### PR TITLE
Add expected invoices count to import batch payload

### PR DESCRIPTION
**Pull request by dcarneiro**
This PR updates the docs to reflect the changes added on https://github.com/socialpaymentsbv/billing-engine/pull/9454

### CODE REVIEW

**Review by vruizext — APPROVED** [review]

File: api-documentation/import.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ Import successfully retrieved.
   "prepaid_amount_currency": "EUR|GBP|CHF",
   "settled_amount_cents": 456,
   "settled_amount_currency": "EUR|GBP|CHF",
+  "expected_invoices_count": 789,
```
Comment (vruizext) [inline api-documentation/import.md]: Here there are only 2 invoices returned, shouldn't `expected_invoices_count` be `2` ?

```suggestion
  "expected_invoices_count": 2,
```

**Review by cristianbica — APPROVED** [review]
:+1:

File: api-documentation/import.md (hunk 1 above)
Comment (dcarneiro) [reply api-documentation/import.md]: If the request is correctly done, yes. I'll update it

Comment (dcarneiro) [reply api-documentation/import.md]: We are actually returning 3 invoices here, not 2

**Pull request merged**